src/redisCache.ts

```diff
@@ -0,0 +1,42 @@
+import redis from 'redis';
+import logger from '@/logger';
+import config from '../config.json';
+
+const { host, port } = config.redis;
+
+export const redisClient = redis.createClient({ socket: { host, port } });
```

Comment:
```suggestion
export const redisClient = createClient({ socket: { host, port } });
```
Update the function call since we changed the import.